### DISCUSSION TITLE
Account (not this) got suspended

### DISCUSSION THREAD

**Discussion by still-hash-ir** [community/community#134510] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi GitHub Community,

My account got suspended last week. I created a new account (this) to access this page and start a discussion. The username of my account that got suspended is `hash-ir`. At first, my profile and repositories were not visible publicly. While I still had the access, I submitted a ticket (#2903190) to request an appeal and reinstatement on Fri, July 26. A day later, the account got suspended. Unfortunately, there has been no response to the ticket yet. 

I really need access to my account for working on professional and personal projects. I am also in the middle of a job search and need to share my GitHub page with the recruiters. 

I understand the importance of adhering to GitHub’s Terms of Service and Community Guidelines. If there were any inadvertent violations on my part, I sincerely apologize and am committed to ensuring full compliance in the future.

Please review my case and consider reinstating my account. If further information is needed, I am more than willing to provide it.

Thank you for your understanding and assistance.

Sincerely,
Hashir

GitHub Username: hash-ir
Email: [hashirahmad@zhcet.ac.in](mailto:hashirahmad@zhcet.ac.in)

**Comment by alanzimm85** [comment]
I have the same problem, could you resolve it?

**Reply by Myla26** [reply]
same here I'm hoping it get resolve ASAP

**Comment by still-hash-ir** [comment, marked as answer]
Hi,

My account has finally been restored. Support said that their team is currently experiencing a higher volume of inquiries than usual, which has resulted in longer response times. They have been investigating suspicious activity and account hijacking. As a security measure, they have placed restrictions on some accounts to combat this.
